src/julia-syntax.scm

```diff
@@ -335,11 +335,11 @@
          (optional-positional-defs name sparams req opt dfl body
                                    (append req opt vararg) rett)))))
    ;; no optional positional args
-   (let ((names (map car sparams))
-         (anames (map (lambda (x) (if (underscore-symbol? x) UNUSED x))
-                      (llist-vars argl))))
-     (if (has-dups (filter (lambda (x) (not (eq? x UNUSED))) anames))
-         (error "function argument names not unique"))
+   (let* ((names (map car sparams))
+         (anames (map (lambda (x) (if (underscore-symbol? x) UNUSED x))(llist-vars argl)))
```

Comment:
```suggestion
         (anames (map (lambda (x) (if (underscore-symbol? x) UNUSED x)) (llist-vars argl)))
```
Plus the indentation is still wrong --- the start of each `(var value)` pair in the let* should line up. Any good code editor should have a scheme mode that does this automatically.

Reply:
Sorry, was using vi to edit the file. Used sublime editor to do the indentation automatically and added the space in suggestion in 02a087a